test/e2e-go/features/participation/onlineOfflineParticipation_test.go

```diff
@@ -290,7 +312,22 @@ func TestAccountGoesOnlineForShortPeriod(t *testing.T) {
 	a.NoError(err, "rest client should be able to add participation key to new account")
 	a.Equal(newAccount, partkeyResponse.Parent.String(), "partkey response should echo queried account")
 	// account uses part key to go online
-	goOnlineTx, err := client.MakeUnsignedGoOnlineTx(newAccount, &partkeyResponse, partKeyFirstValid, partKeyLastValid, transactionFee, [32]byte{})
+	goOnlineTx := partkeyResponse.GenerateRegistrationTransaction(
```

Comment:
Do you think it makes sense to wrap this logic in a new LibGoalClient function and use it for the tests that used to use `client.MakeUnsignedGoOnlineTx`?